_posts/bkk16/2016-02-22-bkk16-100.markdown

```diff
@@ -1,6 +1,6 @@
 ---
-amazon_s3_presentation_url: https://s3.amazonaws.com/connect.linaro.org/bkk16/Presentations/Monday/BKK16-100.pdf
-amazon_s3_video_url: https://s3.amazonaws.com/connect.linaro.org/bkk16/Videos/Monday/BKK16-100%20Evolution%20of%20the%20Reference%20Software.mp4
+amazon_s3_presentation_url: http://connect.linaro.org.s3.amazonaws.com/bkk16/Presentations/Monday/BKK16-100.pdf
```

Comment:
Shouldn't this be https?

Reply:
It appears that https is **not** supported:
https://connect.linaro.orgs.3.amazonaws.com/bkk16/Presentations/Monday/BKK16-100.pdf

Reply:
@pcolmer bucket names that contain periods don't support ssl..
https://stackoverflow.com/questions/62568657/ssl-certificate-issue-with-bucket-name-containing-dots-while-trying-to-use

Reply:
But you're right.. this does actually cause a Mixed Content error in the dev console...
```
Mixed Content: The page at 'https://production-connect-linaro-org-756.websitepreview.linaro.org/resources/bkk16/bkk16-507-aosp-builds-linaro-ci-v2/' was loaded over HTTPS, but requested an insecure frame 'http://connect.linaro.org.s3.amazonaws.com/bkk16/Presentations/Friday/BKK16-507.pdf'. This request has been blocked; the content must be served over HTTPS.
```
So either we need to add a CF distribution in front of the bucket or we migrate resources over to the static-linaro-org bucket.

Reply:
Yeah, this is why we need to migrate the resources over :)